File: dev/test_data/main/csharp/expected/implementation_specific/input/snippets/Verification/texts_are_unique.cs
/// <summary>
/// Check that the <paramref name="texts" /> do not repeat.
/// </summary>
public static bool TextsAreUnique(IEnumerable<string> texts)
{
    var textSet = new HashSet<string>();
    foreach (var text in texts)
    {
        if (textSet.Contains(text))
        {
            return false;
        }
        textSet.Add(text);
    }
    return true;
}

File: dev/test_data/main/python/expected/implementation_specific/input/snippets/Verification/texts_are_unique.py
def texts_are_unique(texts: Iterable[str]) -> bool:
    """Check that the :paramref:`texts` do not repeat."""
    text_set = set()  # type: Set[str]
    for text in texts:
        if text in text_set:
            return False

        text_set.add(text)

    return True
